### ISSUE TITLE
Clicking ”Yes” for saving a phrase doesn't save the phrase

### ISSUE THREAD

**Issue by GoogleCodeExporter** [eadevlin/autokey#29]
```
What steps will reproduce the problem?
1. Edit a phrase
2. Without saving, click another phrase
3. A dialogue pops up asking if you want to save the curretn phrase first
4. Click ”Yes”

What is the expected output? What do you see instead?
The current phrase should be saved and I can start fiddling with the other 
phrase. Instead the current phrase is NOT saved, and I have to save it 
manually.

What version of the product are you using? On what operating system?
AutoKey for GTK 0.61.5 on Ubuntu 9.10.


Please provide any additional information below.
Nothing that I can think of, sorry.

Regards

Johnny Rosenberg

```

Original issue reported on code.google.com by `gurus.kn...@gmail.com` on 25 Mar 2010 at 6:10

**Comment by GoogleCodeExporter** [comment]
```
This is annoying... I don't know why the event is being thrown away after the 
user
clicks yes. I suspect a bug in GTK (yet again). Will try to fix at some point
```

Original comment by `cdekter` on 30 Mar 2010 at 11:23
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
It does actually save the changes, just the selection isn't changed. GTK 
treewidget
ugliness...
```

Original comment by `cdekter` on 12 Apr 2010 at 7:37

**Comment by GoogleCodeExporter** [comment]
```
Fixed in v0.70.1
```

Original comment by `cdekter` on 13 Apr 2010 at 8:21
- Changed state: **Fixed**
